### PR TITLE
fix application checkbox logic

### PR DESCRIPTION
**Pull request by j1joey**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

# What is the ticket # detailing the issue?

Ticket: [ME-185069449](https://www.pivotaltracker.com/n/projects/2640060/stories/185069449)

# A brief description of the changes

Current behavior: We currently only need to fill out the signature box in order to submit an application

New behavior: We now will have to fill out the signature box as well as check all the checkboxes to submit an application.

# Environment Variable

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. Please share the name of the variable below that would enable/disable the feature and which client it applies to.

Variable name:

- [ ] DC
- [ ] ME

### CODE REVIEW

File: features/support/object_model_pages/ivl/iap/ivl_iap_other_questions_page.rb
Comment (jacobkagon) [inline features/support/object_model_pages/ivl/iap/ivl_iap_other_questions_page.rb:1]: Why did you add this? Are you using it anywhere?

Comment (j1joey) [reply features/support/object_model_pages/ivl/iap/ivl_iap_other_questions_page.rb:1]: I had been using them for the spec at first but dont use them anymore, I'll remove them from the pr

File: components/financial_assistance/app/assets/javascripts/financial_assistance/step_conditionals.js [hunk 1]
```diff
@@ -531,31 +531,40 @@ $(document).ready(function(){
...
     first_name_thank_you = $("#first_name_thank_you").val() ? $("#first_name_thank_you").val().toString().toLowerCase().trim() : '';
     last_name_thank_you = $("#last_name_thank_you").val() ? $("#last_name_thank_you").val().toString().toLowerCase().trim() : '';
     subscriber_first_name = $("#subscriber_first_name").val();
     subscriber_last_name = $("#subscriber_last_name").val();
-
-    if(last_name_thank_you == ""){
-      $('.interaction-click-control-submit-application').addClass('disabled');
-    }
-    if( first_name_thank_you == subscriber_first_name && last_name_thank_you == subscriber_last_name){
+    living_outside_no = $('#living_outside_no').is(':checked');
```
Comment (jacobkagon) [inline components/financial_assistance/app/assets/javascripts/financial_assistance/step_conditionals.js:540]: @j1joey JS variables have the term var, let, or const before them. So you could do `var living_outside_no  =  $('#living_outside_no').is(':checked');`

**Review by jacobkagon — CHANGES_REQUESTED** [review]

**Review by jacobkagon — APPROVED** [review]
Changes look good. Please promote for testing. Nice job @j1joey

**Review by jayreddy519 — APPROVED** [review]
Looks good to me.
